### DISCUSSION TITLE
command 'dbtPowerUser.<any command>' not found

### DISCUSSION THREAD

**Discussion by alittlesliceoftom** [AltimateAI/vscode-dbt-power-user#401] [General]
### Expected behavior

When I run ctrl + # then I get compiled code, ctrl + enter -> code execute etc.  Also tried with vscode command dispatch at ctrl shift p

### Actual behavior

When I try and run any power user commands I get an error of format: 

![image](https://user-images.githubusercontent.com/11210917/206237027-d3d71518-aafc-41b4-bcad-26453be44ee3.png)

And the extension doesn't detect the children and parents either

### Steps To Reproduce

Run any command. 

### Operating System

Windows 10

### dbt version

dbt=1.3.0 (also 1.1 - tried both)

### dbt Adapter

dbt-synapse

### dbt Power User version

0.10.1

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by puruzio** [comment]
Hi @alittlesliceoftom, have you been able to figure this one out? I have been having this issue, and would really like to find a solution. In my case, it seems to be some Windows security setting that causes this issue because the extension works fine in one machine outside of Citrix, but not the other one inside Citrix. The problem is that the error message in the Browser console is not clear about the cause.

```
notificationsAlerts.ts:40 Error: command 'dbtPowerUser.testCurrentModel' not found
    at S._tryExecuteCommand (commandService.ts:91:26)
    at S.executeCommand (commandService.ts:85:15)
    at async M.runAction (actions.ts:192:3)
    at async M.run (actions.ts:183:4)
    at async g.onClick (menuEntryActionViewItem.ts:152:4)
onDidChangeNotification	@	notificationsAlerts.ts:40
(anonymous)	@	notificationsAlerts.ts:28
invoke	@	event.ts:626
deliver	@	event.ts:828
fire	@	event.ts:789
addNotification	@	notifications.ts:204
error	@	notificationService.ts:130
(anonymous)	@	titleControl.ts:204
invoke	@	event.ts:626
deliver	@	event.ts:828
fire	@	event.ts:789
run	@	actions.ts:188
await in run (async)		
run	@	titleControl.ts:83
onClick	@	menuEntryActionViewItem.ts:152
(anonymous)	@	actionViewItems.ts:149
```
I was previously told by one of the contributors that this happens because the `dbt-project.yml` is either bad or not found in the workspace, but I don't see how that's the case.

Thanks.

**Comment by anandgupta42** [comment]
@alittlesliceoftom Are you still experiencing the issue?

**Comment by felipesabate-pullpodev** [comment]
Hi team,

This issue seems to have returned in the latest versions of the extension.
I'm experiencing the exact same problem described in [#401](https://github.com/innoverio/vscode-dbt-power-user/issues/401):

Problem:
Whenever I try to run any dbt Power User command (e.g., Execute SQL, Compile SQL, Run Current Model), I get the error:

bash
Copy
Edit
command 'dbtPowerUser.<any command>' not found
Details:

Extension version: 0.58.7

OS: Windows 10

VS Code version: (add your VS Code version here)

dbt version: 1.9.4

Adapter: BigQuery

Python environment: Virtual environment (venv)

What I tried:

Verified dbt works fine in the terminal (dbt run works as expected).

Confirmed dbt_project.yml exists in the root of the opened folder.

Set dbt Path, Profiles Dir, and Project Dir in extension settings.

Restarted VS Code multiple times.

Reinstalled the extension.

Behavior:

The extension loads, but none of its commands are available.

Parent/child detection also does not work.

Expected behavior:

Commands like Execute SQL or Compile SQL should work from the editor or Command Palette.

Steps to reproduce:

Open a dbt project in VS Code on Windows.

Try any dbt Power User command via Command Palette or shortcut.

Please let me know if there is any additional logging I can provide.
Thanks for looking into this — it’s a great extension and super helpful when working!

**Reply by HansHuller** [reply]
Suffering from the same issue. Please post here if you find any fixes.

_Edit: It just went back to working normally, I didn't take any actions on my end._

**Comment by felipesabate-pullpodev** [comment]
Hi Hans,

It suddenly started working for me without any changes on my side.
I did notice there was an update released on July 31st, so that might have
fixed it.
